pkg/model/firewall.go

```diff
@@ -240,6 +240,7 @@ func (b *FirewallModelBuilder) applyNodeToMasterBlockSpecificPorts(c *fi.ModelBu
 	// Don't allow nodes to access etcd peer port
 	tcpBlocked[2380] = true
 	tcpBlocked[2381] = true
+	tcpBlocked[2382] = true
```

Comment:
Could we conditionally block this port only if the cluster is using the etcd managed cilium? that would remove the security group changes being reported for clusters that don't use etcd-managed cilium which might be preferable. 

There are other CNIs conditionally blocking ports below, so it might be wise to do that in this case as well.